views/tables/ParticipantsTable/index.jsx

```diff
@@ -63,25 +63,46 @@ const ParticipantsTable = (props) => {
   ]
   const cellRenderer = (participant, property) => {
     switch (property) {
+      case 'days':
+        if (participant?.synced && participant?.Consent) {
+          const lastSyncedDate = dayjs(participant.synced)
+          const consentDate = dayjs(participant.Consent)
+
+          return `${lastSyncedDate.diff(consentDate, 'd')} days`
+        } else if (
+          !participant?.synced &&
+          participant?.Consent &&
+          dayjs(participant.Consent).isValid()
+        ) {
+          const consentDate = dayjs(participant.Consent)
+
+          return `${dayjs().diff(consentDate, 'd')} days`
+        }
+
+        return ''
       case 'synced':
-        const isToday = dayjs(participant[property]).isToday()
-        const isYesterday = dayjs(participant[property]).isYesterday()
-        const isTodayOrYesterday = isToday || isYesterday
-        return (
-          <Typography
-            sx={{
-              color: isTodayOrYesterday ? 'green.500' : 'text.primary',
-            }}
-          >
-            {isToday
-              ? 'Today'
-              : isYesterday
-              ? 'Yesterday'
-              : participant[property]
-              ? dayjs(participant[property]).format('MMM-D-YYYY')
-              : ''}
-          </Typography>
-        )
+        if (participant[property] && dayjs(participant[property]).isValid()) {
```

Comment:
what do you think about extracting `dayjs(participant[property])` to a constant (`participantSyncDate` or something like that) since we reference it multiple times?

Reply:
[0af550e](https://github.com/AMP-SCZ/dpdash/pull/600/commits/0af550ebd947fb7c731cba9479f9382fa5ea9138)

Reply:
@Elkrival this commit does not address the comment.

Reply:
Thanks for that. 
[a3b01fe](https://github.com/AMP-SCZ/dpdash/pull/600/commits/a3b01fedffc4a9507fc4aeb4ceb94e6e998ecb48)